cgnet/simplify.py

```diff
@@ -0,0 +1,33 @@
+import networkx as nx
+
+def is_simple_node( graph, node ):
```

Comment:
I don't think that there should be spaces around parameters (see [PEP 8](https://www.python.org/dev/peps/pep-0008/#pet-peeves)).